website/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "my-application",
+	"version": "0.0.1",
+	"scripts": {
+		"dev": "webpack serve --hot --mode development",
+		"build-webpack": "webpack --mode production",
+		"build": "yarn typedoc && yarn build-webpack",
+		"dev-disk": "webpack --mode development --watch",
+		"typedoc": "typedoc --options ./typedoc/typedoc.json"
+	},
+	"dependencies": {
+		"@hediet/std": "0.6.0",
```

Comment:
@hediet It would be nice if this package linked back to a GH repo, with a proper license.